### PR TITLE
Upgrade to JDK 14

### PR DESCRIPTION
**Pull request by Toparvion**
The upgrading to JDK 14 is extracted into separate PR because of compatibility [issue](https://community.sonarsource.com/t/issue-java-14-java-lang-noclassdeffounderror-java-util-jar-pack200/21853/26) in SonarCloud. I hope it will be resolved soon.

To merge this PR we should:

- [x] ensure that SonarCloud works with builds on JDK14
- [x] revert the change in Travis badge (in [README.md](https://github.com/Toparvion/analog/pull/36/files#diff-04c6e90faac2675aa89e2176d2eec7d8R3)) to make it point to `master` branch

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
...
+[![Travis (.com)](https://img.shields.io/travis/com/toparvion/analog/jdk14?style=plastic)](https://travis-ci.com/github/Toparvion/analog) [![Sonar Coverage](https://img.shields.io/sonar/coverage/Toparvion_analog?server=https%3A%2F%2Fsonarcloud.io&style=plastic)](https://sonarcloud.io/dashboard?id=Toparvion_analog) [![Sonar Quality Gate](https://img.shields.io/sonar/quality_gate/Toparvion_analog?server=https%3A%2F%2Fsonarcloud.io&style=plastic)](https://sonarcloud.io/dashboard?id=Toparvion_analog) [![GitHub release (latest SemVer)](https://img.shields.io/github/v/release/toparvion/analog?color=green&sort=semver&style=plastic)](https://github.com/Toparvion/analog/releases/latest) [![GitHub](https://img.shields.io/github/license/toparvion/analog?style=plastic)](https://github.com/Toparvion/analog/blob/master/LICENSE)
```
Comment (Toparvion) [inline README.md]: This is changed just to reflect the current state of the build in the branch (mostly for learning purposes). Therefore it should be reverted before the merge:
```suggestion
[![Travis (.com)](https://img.shields.io/travis/com/toparvion/analog?style=plastic)](https://travis-ci.com/github/Toparvion/analog) [![Sonar Coverage](https://img.shields.io/sonar/coverage/Toparvion_analog?server=https%3A%2F%2Fsonarcloud.io&style=plastic)](https://sonarcloud.io/dashboard?id=Toparvion_analog) [![Sonar Quality Gate](https://img.shields.io/sonar/quality_gate/Toparvion_analog?server=https%3A%2F%2Fsonarcloud.io&style=plastic)](https://sonarcloud.io/dashboard?id=Toparvion_analog) [![GitHub release (latest SemVer)](https://img.shields.io/github/v/release/toparvion/analog?color=green&sort=semver&style=plastic)](https://github.com/Toparvion/analog/releases/latest) [![GitHub](https://img.shields.io/github/license/toparvion/analog?style=plastic)](https://github.com/Toparvion/analog/blob/master/LICENSE) 
```

File: README.md [hunk 2]
```diff
@@ -45,7 +45,7 @@ In AnaLog terms the *administrator* is a person who installs and configures AnaL
 From the administrator's perspective AnaLog:
 
 * is standalone Java application with built-in web server (based on [Spring Boot](https://spring.io/projects/spring-boot) framework)
-* works on [Java 12](http://jdk.java.net/12/) and above
+* works on [Java 13](http://jdk.java.net/13/) and above
```
Comment (a-polyudov) [inline README.md]: 14?

**Comment by Toparvion** [comment]
Now SonarCloud does not break the build though it still doesn't recognize JDK 14 bytecode, see [this](https://travis-ci.com/github/Toparvion/analog/builds/164963639) build log:
```text
Caused by: java.lang.IllegalArgumentException: Unsupported class file major version 58
	at org.objectweb.asm.ClassReader.<init>(ClassReader.java:195)
	at org.objectweb.asm.ClassReader.<init>(ClassReader.java:176)
	at org.objectweb.asm.ClassReader.<init>(ClassReader.java:162)
	at org.sonar.java.bytecode.se.MethodLookup.lookup(MethodLookup.java:68)
	at org.sonar.java.bytecode.se.MethodLookup.lookup(MethodLookup.java:60)
	at org.sonar.java.bytecode.se.BytecodeEGWalker.execute(BytecodeEGWalker.java:316)
	at org.sonar.java.bytecode.se.BytecodeEGWalker.getMethodBehavior(BytecodeEGWalker.java:293)
	... 254 common frames omitted
```
This should be fixed with Sonar [updating](https://jira.sonarsource.com/browse/SONARJAVA-3270) to ASM 8.0.

File: README.md (hunk 2 above)
Comment (Toparvion) [reply README.md]: Right! Fixed with [cd8c3a5](https://github.com/Toparvion/analog/pull/36/commits/cd8c3a5ff198af794941a9fbbd1a0bed0cf8c479) commit.
Thanks for the pointing.

**Pull request merged**